cmd/docker/docker.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 var allowedAliases = map[string]struct{}{
-	"builder": {},
+	"builder": {}, // TODO: builder alias should be deprecated (buildx by default)
```

Comment:
Should we deprecate the builder alias?

Reply:
Only about 40 people a day use it on desktop, so I would say yes.

Reply:
@thaJeztah It's about the _alias builder_ here through `~/.docker/config.json`, not the `docker builder` command which is an other impl.

Reply:
Ah! Looks like I misunderstood; it's to remove it because the alias is now  there by default

Reply:
I think we should leave it for compatibility and so we have upgrade path that is hardcoded like today.